### PR TITLE
Improve styling and simplfy workflow example

### PR DESCRIPTION
**Pull request by planger**
https://github.com/eclipse-glsp/glsp/issues/492

### CODE REVIEW

File: packages/client/css/glsp-sprotty.css [hunk 1]
```diff
@@ -50,23 +51,59 @@
...
+.sprotty-node:not(.selected) {
+    stroke-width: 0px;
+}
+
+.sprotty-node.mouseover:not(.selected),
+.sprotty-edge.mouseover:not(.selected) {
+    stroke: #1d1e1f;
+    stroke-width: 1.5px;
+    stroke-dashoffset: 5;
+    stroke-dasharray: 5, 5;
```
Comment (ndoschek) [inline packages/client/css/glsp-sprotty.css:90]: In general, I like the visual differentiation for selection/mouseover via dashed stroke. But due to the dashed line, edges might flicker on mouseover (see also screenshot). Maybe we could think of a solid line in a contrast color?

glsp-example-dashed-flickering

**Review by ndoschek — COMMENTED** [review]
Thanks Philip! I like the improved styling of the workflow example! :tada: 
I spotted one potential issue with the dashed strokes, please see inline.

File: packages/client/css/glsp-sprotty.css (hunk 1 above)
Comment (planger) [reply packages/client/css/glsp-sprotty.css]: Thanks Nina, you are right! We typically fix those issues by adding a transparent wider edge above the visible edge, which helps selection the edge too.
However, we also want to stay very generic, so probably dashes aren't a good choice for hover feedback.
I replaced it with opacity for now.
What do you think?
Also feel free to suggest different colors for the nodes if you don't like them.

**Review by ndoschek — CHANGES_REQUESTED** [review]
Thanks Philip! The selection using opacity looks great!

I played around with the colors a bit, and using complementary colors for the nodes would look great to me, e.g. some darker orange (e.g. #db8651) and some teal (e.g. #5b9fa8).
This would also have the benefit that the blue selection color would give bit more contrast to both colors imo.
Also, I would suggest to use `var(--theia-focusBorder)` instead of `var(--theia-menu-selectionBackground)` as selection color, as it a bit brighter in dark mode as well.

Proposal in light theme:
image

Proposal in dark theme:
image

Also, I noticed two other issues:
- The weighted edges are not colored differently anymore
- The styling of the workflow standalone example: i.e. the resize handles differ from the selection color. Should we align that as well?

image

**Comment by planger** [comment]
Thanks @ndoschek!
I used your suggested colors, added different color for weighted edges, tested for color blindness (see below), used theia color for selection and handles in standalone.

Red-Weak/Protanomaly:
image

I think it still is nicely distinguishable.

**Review by ndoschek — CHANGES_REQUESTED** [review]
Thanks a lot Philip, looks great to me! :tada: 

Could you please update the file headers with the current year here as well, thanks!

**Comment by planger** [comment]
Sure, added 4495c41. Thanks!

**Review by ndoschek — APPROVED** [review]

**Pull request merged**